Resources/public/javascript/cropper.js

```diff
@@ -0,0 +1,24 @@
+let cropTrigger;
+
+class cropSave {
+    constructor(item) {
+        this.item = item
+        cropTrigger = $('*[data-method="getCroppedCanvas"]');
+        cropTrigger.on('click', this.save.bind(this));
+    } 
+    save() {
+    	setTimeout(function() {
+    		$('*[name="btn_update_and_edit"]').click();
+    	}, 500);
+        
+    }
+}
+
+$(function(){
+    $('.cropper').each(function(index,item){
+        new cropSave($(item));
+    });
+});
+
+
+
```

Comment:
Why use an entire class if the only thing it does it fire **$('*[name="btn_update_and_edit"]').click();**? Just doing this should work too:

$('*[data-method="getCroppedCanvas"]').on('click', function() {
    $('*[name="btn_update_and_edit"]').click();
});